web/services/management/commands/removeolddata.py

```diff
@@ -0,0 +1,17 @@
+from django.core.management.base import BaseCommand
+from services.models import ErrorReport
+from datetime import timedelta
+from django.utils import timezone
+
+
+class Command(BaseCommand):
+    help = 'Deletes all recovery files over a month old'
+
+    def handle(self, *args, **options):
+        reports = ErrorReport.objects.filter(
+            dateTime__lte=timezone.now() - timedelta(days=30)).filter(
```

Comment:
Could we put this "magic"  number as a module-level variable, e.g. `DEFAULT_RECOVERY_FILE_AGE_DAYS`?